lib/main_common.pm

```diff
@@ -52,6 +52,8 @@ our @EXPORT = qw(
   load_filesystem_tests
   load_wicked_tests
   load_nfv_tests
+  load_nfv_master_tests
```

Comment:
do you plan also to run this tests in opensuse ?

Reply:
no, just SLE

Reply:
hm , in this case why you adding functions to main_common ? you can just keep them in sle/main.pm . Or even just drop them and write something like this :

```
  boot_hdd_image();
  if (check_var("NFV", "master")) {
        loadtest "nfv/openvswitch_dpdk";
       loadtest "nfv/vsperf_installation";
    }
    elsif (check_var("NFV", "trafficgen")) {
       loadtest "nfv/moongen_installation";   
    }
```

Reply:
@okurz  WDYT ? You insisting to move this section to main_common and I actually support you in this. But for me the fact that this code meant to run only on SLE change my position. What about you ?

Reply:
can we just keep it like this for now and modify later if needed? opensuse is not in the scope for now, but it might be interesting to run this also on opensuse in the future.